install-cloud.sh

```diff
@@ -11,27 +11,28 @@
 
 set -e
 
-if [ -f /etc/centos-release ]
-then 
-  yum update -y
-  yum install -y git bind-utils nmap-ncat
```

Comment:
Yeah, fair enough.  How about this?
We do tests for the commands we'll need, then just abort and ask them to install them?  That's distro-independent.  E.g., git, nmap, etc.